app/components/Status.js

```diff
@@ -3,8 +3,8 @@ import React from 'react';
 class Status extends React.Component {
   render() {
     return (
-      <div className="status"> 
-        <a href="#" onClick={() => {pdfFunctions.openURL(this.props.url)}}>{this.props.title}</a> - {this.props.validUrl?<i className="fa fa-check greenIconColor" aria-hidden="true"> </i>:<i className="fa fa-ban redIconColor" aria-hidden="true"> </i>} 
+      <div className="status unselectable"> 
+        <a href="#" onClick={() => {pdfFunctions.openURL(this.props.url)}}>{this.props.title}</a> - {this.props.validUrl?<i data-tooltip='Valid Link' className="fa fa-check greenIconColor" aria-hidden="true"> </i>:<i data-tooltip='Invalid Link' className="fa fa-ban redIconColor" aria-hidden="true"> </i>} 
```

Comment:
```this.props.validUrl?<i data-tooltip='Valid Link' className="fa fa-check greenIconColor" aria-hidden="true"> </i>:<i data-tooltip='Invalid Link' className="fa fa-ban redIconColor" aria-hidden="true"> </i>``` 
This could be:
```
<i data-tooltip={this.props.valudUrl ? 'Valid Link' : 'Invalid Link'} className={this.props.validUrl ? 'fa fa-check greenIconColor' : 'fa fa-ban redIconColor'} aria-hidden="true"> </i>
```
And I think maybe this is more readable